views/status.tmpl

```diff
@@ -8,6 +8,7 @@
     <div class="container" data-controller="status">
         <br/>
         <div class="alert alert-info">
+            {{$P2PKHAddr := PKAddr2PKHAddr .AdditionalInfo}}
```

Comment:
I think this should be done in the ` {{else if eq .StatusType "P2PK Address Type"}}` block where it applies only.